CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.8.0 (March 26th, 2019)
```

Comment:
Can you change the date here to "UNRELEASED" so we know to go back and update it when the actual release is cut please?

Reply:
Actually I grab this when updating the change log before the release is finalized, so we're good here. 👍